src/panels/config/devices/device-detail/integration-elements/zwave_js/ha-device-info-zwave_js.ts

```diff
@@ -117,12 +118,20 @@ export class HaDeviceInfoZWaveJS extends LitElement {
           : this.hass.localize("ui.common.no")}
       </div>
       <div>
-        ${this.hass.localize("ui.panel.config.zwave_js.device_info.is_secure")}:
-        ${this._node.is_secure === true
-          ? this.hass.localize("ui.common.yes")
-          : this._node.is_secure === false
-          ? this.hass.localize("ui.common.no")
-          : this.hass.localize("ui.panel.config.zwave_js.device_info.unknown")}
+        ${this.hass.localize(
+          "ui.panel.config.zwave_js.device_info.highest_security"
+        )}:
+        ${this._node.highest_security_class !== null
+          ? this.hass.localize(
+              `ui.panel.config.zwave_js.highest_security.${
+                securityClass[this._node.highest_security_class]
```

Comment:
`SecurityClass[this._node.highest_security_clas]`

Reply:
well I'll be, I had no idea this is how enums worked